cip-drep-metadata/README.md

```diff
@@ -0,0 +1,241 @@
+
+Participants at the workshop held to debut the first draft of this CIP strongly advocated in favour of a CIP that would include features that would assist a DRep comprised of a team of individuals properly describe their endeavour to delegating Ada Holders and anyone else reading the metadata associated with their registration.
+
+This CIP has not included these features, the decision not to include these features was made in order to simplify this CIP so that it became suitable for the minimum viable level of tooling, with the expectation that further CIPs will build on it. 
+
+### Formatting
+If DRep Metadata is not formatted in the way described by this CIP then tooling SHOULD either:
+1. Deal with it as described in CIP-100 (by rendering the raw unformatted document)
+2. Not display the any of the metadata
+
+In either case the tooling SHOULD provide a warning that this has happened and link to the url described in the metadata anchor of the DRep registration.
+
+### Missing
+If the DRep metadata cannot be found at the address specified in the anchor then the tooling SHOULD provide a warning. There is no need to link to the url in the metadata anchor. 
+
+### Witnesses
+DRep Metadata will not follow the CIP-100 specification related to signing the metadata, the [authors property](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/README.md#high-level-description) can be left blank without the need for tooling providers to warn their users that the author has not been validated. Instead the author can be derived from the DRep ID associated with the registration. The need for an `authors` field will also be discarded in favour of including usernames inside of the `body` field. For the avoidance of doubt this CIP recommends that the entire authors property be left blank, and that tooling ignore it. 
+
+### Extended Vocabulary
+Like CIP-108, this CIP also extends the potential vocabulary of CIP-100's `body` property. For all of these properties tooling providers MUST be aware that they are responsible for what they display to their users and that these fields could be used for illegal, unsavoury, or innapropriate purposes or language. Therefore such tools SHOULD have a terms of service which they enforce to moderate what they show to their users. The following are a list of properties tooling should expect to encounter:
+
+#### `paymentAddress`
+Dreps may want to recieve tokens for a variety of reasons such as:
+- donations
+- expenses
+- any incentive program
+
+Therefore there will be a `payment_address` field in the metadata where such payments could be sent. This makes such an address public and payments to DReps transparent. 
+
+#### `username`
+- Compulsory
+- A very short freeform text field. Limited to 80 characters.
+- This SHOULD NOT support markdown text styling.
```

Comment:
```suggestion
- This MUST NOT support markdown text styling.
```
I would change this to a "MUST NOT" because making it "SHOULD NOT" means that someone inevitably will use markdown or emojis or other "rich elements" that then force all downstream tooling to support them (see the whole SPO Ticker Debacle).

Reply:
yes, we MUST use more emphatic language when there is something that MUST be done (even though we also "should" follow this rule) 🧐